### PR TITLE
Exposing 'initWithPreset' methods for existing prests

### PR DESCRIPTION
**Pull request by narner**

### CODE REVIEW

File: AudioKit/Operations/Signal Generators/Oscillators/AKFMOscillator.h [hunk 1]
```diff
@@ -40,21 +40,39 @@ NS_ASSUME_NONNULL_BEGIN
 + (instancetype)presetDefaultOscillator;
 
 /// Instantiates the oscillator with 'stun-ray' type values
-+ (instancetype)presetStunRay;
++ (instancetype)initWithPresetStunRay;
```
Comment (megastep) [inline AudioKit/Operations/Signal Generators/Oscillators/AKFMOscillator.h]: You probably shouldn't have a class method's name that starts with `init...`, that's pretty much against adopt conventions, and I would expect to create problems with Swift.

**Comment by megastep** [comment]
Also the CI builds fail now. ;)

**Comment by narner** [comment]
Crud, my mistake. I'll fix this in a bit!

**Comment by narner** [comment]
Looks like Travis is happy with me again :)

**Comment by megastep** [comment]
Looks good!

**Pull request merged**